src/common/consumer/consumer.h

```diff
@@ -430,6 +430,13 @@ struct lttng_consumer_stream {
 	pthread_cond_t metadata_rdv;
 	pthread_mutex_t metadata_rdv_lock;
 
+	/*
+	 * If rotate_position != 0, when we reach this position in the
+	 * ring-buffer, close this tracefile and create a new one in
+	 * chan->pathname.
+	 */
```

Comment:
Can we clarify that `rotate_position` is the a position in the ring-buffer?
How about that? :
```
/*
 * rotate_position represents the position in the ring-buffer that has 
 * to be flushed to disk to complete the ongoing rotation. When that
 * position is reached, this tracefile can be closed and a new one is
 * created in char->pathname.
 */
```